extra/mender-gateway/docker-compose.test.yml

```
@@ -0,0 +1,13 @@
+version: '2.1'
+services:
+  mender-gateway:
+    image: registry.mender.io/mendersoftware/mender-gateway:master
```

Note,

This line to be modified after [MEN-5300](https://tracker.mender.io/browse/MEN-5300)
```suggestion
    image: registry.mender.io/mendersoftware/mender-gateway-qemu-commercial:mender-master
```